tests/unit/rss/test_views.py

```diff
@@ -64,5 +67,40 @@ def test_rss_packages(db_request):
     project3.created = datetime.date(2013, 1, 1)
     ReleaseFactory.create(project=project3)
 
-    assert rss.rss_packages(db_request) == {"newest_projects": [project3, project1]}
+    assert rss.rss_packages(db_request) == {
+        "newest_projects": tuple(zip((project3, project1), (None, None)))
+    }
     assert db_request.response.content_type == "text/xml"
+
+
+def test_format_author(db_request):
+    db_request.find_service = pretend.call_recorder(
+        lambda *args, **kwargs: pretend.stub(
+            enabled=False, csp_policy=pretend.stub(), merge=lambda _: None
+        )
+    )
+
+    db_request.session = pretend.stub()
+
+    project = ProjectFactory.create()
+    release = ReleaseFactory.create(project=project)
+
+    release.author = None
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == release.author_email
+
+    release.author = "No Reply"
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == "No Reply <noreply@pypi.org>"
+
+    release.author = "Nön Äscii"
+    release.author_email = "noreply@pypi.org"
+    assert rss._format_author(release) == "Nön Äscii <noreply@pypi.org>"
+
+    release.author = "Author One, Author Two, Author Three"
+    release.author_email = "noreply@pypi.org, noreply@pypi.org, noreply@pypi.org"
+    assert rss._format_author(release) == release.author_email
+
+    release.author = "John Doe"
+    release.author_email = None
+    assert rss._format_author(release) is None
```

Comment:
Nit: is there a reason to not include the author at all if their email is `None`? Seems like we should include that information if we do have it.

Reply:
Content of `<author>` is specified to be the author's email address in RSS, that's why I'm omitting it if we don't have one. http://www.rssboard.org/rss-specification#ltauthorgtSubelementOfLtitemgt

Reply:
Got it, thanks for the link. If we're adhering this closely to the spec it looks like the proper form should be `lawyer@boyer.net (Lawyer Boyer)`, which is slightly different than the current `No Reply <noreply@pypi.org>`.

Not a blocker by any means, I'm just a bit unsure how much we can deviate from the spec or not.

Reply:
I think the formatting in the spec is pretty much just an example. Anyway the current revision of the PR uses the bare email address, and ignores the realname.